src/dmd/globals.d

```diff
@@ -148,6 +148,8 @@ extern (C++) struct Param
     DiagnosticReporting useDeprecated = DiagnosticReporting.inform;  // how use of deprecated features are handled
     bool stackstomp;            // add stack stomping code
     bool useUnitTests;          // generate unittest code
+    bool UnittestRootOnly;      // generate unittest code only for root modules
+    bool UnittestFirstRootOnly; // generate unittest code only for the first root module
```

Comment:
It would appear that this is better as an enum, instead of three mutually exclusive fields.

Reply:
Thanks!